src/app/loadout-drawer/subclass-drawer/SubclassPlugDrawer.tsx

```diff
@@ -235,6 +244,6 @@ function filterAvailablePlugsForProfile(
   );
 
   return dimPlugSet.plugs.filter((plug) =>
-    availablePlugs.some((p) => p.plugItemHash === plug.plugDef.hash)
+    availablePlugs.some((p) => p.plugItemHash === plug.plugDef.hash && p.canInsert)
```

Comment:
Changed this one to only show unlocked plugs as well

Reply:
Cool, this was basically just taking the functionality that was available in the "everywhere" plugset helper and defining it here, since it was the only place using it.